addons/website_event/views/event_templates.xml

```diff
@@ -88,34 +83,38 @@
     </t>
 </template>
 
-<template id="event_right_photos" inherit_id="website_event.index" active="False" customize_show="True" name="Photos">
+<template id="event_right_column" inherit_id="website_event.index" active="False" customize_show="True" name="Right Content Column">
```

Comment:
Not sure about the translation of "Right Content Column", I would use "Content Column on the Right"